src/parser/parse_error.ml

```diff
@@ -41,9 +41,15 @@ let no_trailing_whitespace_in_verbatim : Location.span -> Error.t =
 let only_one_title_allowed : Location.span -> Error.t =
   Error.make "only one title-level heading is allowed"
 
+let page_title_required : string -> Error.t =
+  Error.filename_only "page title is required, add one with {0 ...}"
+
 let sections_not_allowed : Location.span -> Error.t =
   Error.make "sections not allowed in this comment"
 
+let titles_not_allowed : Location.span -> Error.t =
+  Error.make "titles not allowed"
```

Comment:
Perhaps instead of "titles not allowed," this should say something like "titles ('{0 …}') are only allowed in .mld files"

Reply:
Changed it to `level-0 headings (titles) are only allowed in pages`.